### DISCUSSION TITLE
[Bug]: I need guidance

### DISCUSSION THREAD

**Discussion by eylandoo** [wg-easy/wg-easy#1484] [Q&A]
### Describe the bug

I want to install the nightly version
Help me where I should change it in the default settings 
I desperately need to set an expiration date for each task

please 

### Expected behavior

Work just fine!

**Comment by kaaax0815** [comment]
You filled this as an bug. Use Discussion for this. Converting

Please provide more information.

What are you using docker-compose or docker run?

See https://docs.docker.com/engine/containers/run/#image-references and https://docs.docker.com/reference/compose-file/services/#image and https://github.com/wg-easy/wg-easy?tab=readme-ov-file#versions

**Comment by eylandoo** [comment]
I'm pre-installing Docker 
But I just want to leave it for my users to expire, tell me how to install it because it's not in the regular version

**Reply by kaaax0815** [reply]
Client Expiry is currently only available in the `nightly` tag
